freeglut/freeglut/CMakeLists.txt

```diff
@@ -391,12 +396,22 @@ ENDIF()
 
 SET(LIBNAME freeglut)
 IF(WIN32)
+    IF(FREEGLUT_REPLACE_GLUT)
+        SET(LIBNAME glut)
+    ELSE()
+        SET(LIBNAME freeglut)
+    ENDIF()
+
     LIST(APPEND LIBS winmm)
     IF(FREEGLUT_BUILD_SHARED_LIBS)
         TARGET_COMPILE_DEFINITIONS(freeglut PRIVATE FREEGLUT_EXPORTS)
+        SET_TARGET_PROPERTIES(freeglut PROPERTIES OUTPUT_NAME ${LIBNAME})
     ENDIF()
     IF(FREEGLUT_BUILD_STATIC_LIBS)
         TARGET_COMPILE_DEFINITIONS(freeglut_static PUBLIC FREEGLUT_STATIC)
+        IF(FREEGLUT_REPLACE_GLUT AND NOT FREEGLUT_BUILD_SHARED_LIBS)
+            SET_TARGET_PROPERTIES(freeglut_static PROPERTIES OUTPUT_NAME ${LIBNAME})
```

Comment:
Thanks for this! One question here. If i read this correctly, this causes the static build to be called glut, in the case only static build is requested. Is that a valid way to replace glut (i.e., won't lead to trouble)?